docker-compose.yml

```diff
@@ -58,15 +81,15 @@ services:
       dockerfile: Dockerfile
     volumes:
       # Docker socket mounting is required to restart the socks5 proxy container from inside this container
-      - /var/run/docker.sock:/var/run/docker.sock
+      # - /var/run/docker.sock:/var/run/docker.sock
       - ./local/logs:/youtube-synch/local/logs
       - ./local/data:/youtube-synch/local/data
       # mount Google Cloud's Application Default Credentials file. A default bind mount is created
       # as workaround for scenario when `YT_SYNCH__YOUTUBE__ADC_KEY_FILE_PATH` will be undefined,
       # since docker-compose does not support creating bind-mount volume with empty path.
       - ${YT_SYNCH__YOUTUBE__ADC_KEY_FILE_PATH:-./package.json}:${YT_SYNCH__YOUTUBE__ADC_KEY_FILE_PATH:-/package.json}:ro
       # mount the AWS credentials file to docker container home directory
-      - ~/.aws/:/root/.aws:ro
+      # - ~/.aws/:/root/.aws:ro
```

Comment:
The AWS credentials are loaded from .env? That is why we don't need to mount the aws folder?

Reply:
Yes